user_docs/en/userGuide.md

```diff
@@ -2297,6 +2297,12 @@ If this is enabled, NVDA will automatically speak the character at the cursor wh
 
 To toggle this option from anywhere, please assign a custom gesture using the [Input Gestures dialog](#InputGestures).
 
+##### Speak when navigating by line or paragraph {#BrailleSpeakOnNavigating}
+
+If this is enabled, NVDA will speak the reached line or paragraph when moving the braille display to the previous or next line.
```

Comment:
Using "reached" with this sentence structure, sounds a bit irregular in English. Maybe try:

```suggestion
Enabling this option will cause NVDA to speak lines or paragraphs reached using the braille display's navigation controls.
```

At least, given my understanding of the option as essentially "speech follows braille".